lib/time.js

```diff
@@ -152,17 +152,32 @@ function CronTime(luxon) {
 			}
 
 			if (typeof this.utcOffset !== 'undefined') {
-				let offset =
+				if (typeof this.utcOffset === 'string') {
+					const parts = this.utcOffset.split(':');
+					const hours = parseInt(parts[0]);
+					const minutes = parseInt(parts[1] ? parts[1] : '0');
+					if (this.utcOffset[0] === '-') this.utcOffset = hours * 60 - minutes;
+					else this.utcOffset = hours * 60 + minutes;
+				}
+
+				let offsetHours =
 					this.utcOffset >= 60 || this.utcOffset <= -60
 						? this.utcOffset / 60
 						: this.utcOffset;
-				offset = parseInt(offset);
+
+				offsetHours = parseInt(offsetHours);
+
+				const offsetMins = Math.abs(this.utcOffset - offsetHours * 60);
+				const offsetMinsStr = offsetMins >= 10 ? offsetMins : '0' + offsetMins;
```

Comment:
when defining `offsetHours`  the time unit of `this.utcOffset` (hours or minutes) is subject to a condition, but when defining `offsetMins` (which depends `offsetHours`), that condition isn't present.
This makes the `should run a job using cron syntax with numeric format utcOffset with minute` test fail with the following values:
```
this.utcOffset: 15
offsetHours: 15
offsetMins: 885 [ Math.abs(15−15×60) ]
utcZone: UTC+15:885 => invalid UTC offset
```

Please remember to run `npm run test && npm run lint -- --fix` before you commit :wink: 
*there is also a minor codestyle issue that will be fixed by the above command*